docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_position: 4
+title: Cryptographic verification of equivocation evidence
+---
+# ADR 005: Cryptographic verification of equivocation evidence
```

Comment:
It seems the there's a more [recent version](https://github.com/cosmos/interchain-security/blob/main/docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md) of this. Does it make sense to use this one?